test/SlowTests/Authentication/AuthenticationLetsEncryptTests.cs

```diff
@@ -31,7 +31,7 @@ public AuthenticationLetsEncryptTests(ITestOutputHelper output) : base(output)
         {
         }
         
-        [RetryFact(delayBetweenRetriesMs: 1000)]
+        [RavenIntegrationRetryFact(delayBetweenRetriesMs: 1000)]
```

Comment:
Please adjust https://jenkins.hibernatingrhinos.com/job/PR_Tests/10123/testReport/junit/SlowTests.Tests/TestsInheritanceTests/AllTestsShouldUseRavenFactOrRavenTheoryAttributes/ test because of that change

Reply:
Adjusted.